src/modules/inventory/components/CustomFields.tsx

```diff
@@ -0,0 +1,440 @@
+/* eslint-disable react/prop-types */
```

Comment:
למה ההערה פה? אם יש בעיה עם eslint, צריך לטפל בה

Reply:
?

Reply:
אוקי אוריד ואנסה לטפל

Reply:
אוקי - הורדתי ואני אנסה לסדר פה

Reply:
כשחיפשתי אודות הבעיות שהיו לי עם eslint קבלתי את זה:
The errors you’re encountering are related to eslint's react/prop-types rule. This rule is used to enforce type checking on the props of React components when using JavaScript. However, since you are using TypeScript, you don't need to use prop-types for runtime type checking as TypeScript provides compile-time type checking
בעצם נשמע שמיותר הeslint כשכותבים עם ts?

Reply:
מה שהוצע לי לעשות זה להוסיף את השורה הזאת - 'react/prop-types': 'off',
בקובץ הeslintrc וזה עזר
בסדר?